<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectVideoButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>29</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectOutputFolderButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoPathLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File path</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputPathLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder path</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frequencySpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="formatComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame gap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Formats</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clearFolderCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear old frames from folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideo_to_Picture">
    <property name="title">
     <string>Video Deconstruct</string>
    </property>
   </widget>
   <addaction name="menuVideo_to_Picture"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
